app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_filter :require_user
```

Comment:
I would recommend using `before_action` as filter is going to be deprecated in Rails 5.